src/map/handler/Map.TouchZoom.js

```diff
@@ -109,6 +109,15 @@ L.Map.TouchZoom = L.Handler.extend({
 		    .off(document, 'touchmove', this._onTouchMove)
 		    .off(document, 'touchend', this._onTouchEnd);
 
+		var snap = L.Browser.any3d ? this._map.options.zoomSnap : 1;
+		if (snap) {
```

Comment:
Why are you changing the touch zoom behaviour in the same pull request with a scrollwheel change? If there is an issue about touch-zoom snapping, do provide a bug number for it, and split the change into another commit & pull req.